### PR TITLE
Add optional `redraw` prop

### PR DESCRIPTION
**Pull request by jagged3dge**
re-submitting PR #34 with changes as suggested

### CODE REVIEW

File: src/CountUp.jsx [hunk 1]
```diff
@@ -35,6 +36,12 @@ class CountUp extends React.Component {
     this.startAnimation();
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    return nextProps.redraw
+      ? true
+      : shallowCompare(this, nextProps, nextState);
```
Comment (glennreyes) [inline src/CountUp.jsx]: Lets do:
```js
return nextProps.redraw || shallowCompare(this, nextProps, nextState);
```

instead of the ternary operation. 👍

**Comment by jagged3dge** [comment]
Should this make for a minor release bump due to adding the new dependency?

**Comment by glennreyes** [comment]
I've done the minor bump already 👍

**Comment by jagged3dge** [comment]
Cheers! :beer:

**Pull request merged**